tier1-icons.html

```diff
@@ -176,7 +176,7 @@
         <g id="folder">
```

Comment:
Objects that are recognizable (folders, magnifying glasses, cameras, printers, etc.) should not be mirrored -- a folder is still a folder.